src/pages/background/resource/signin/index.ts

```diff
@@ -31,15 +31,27 @@ export const updateSignins = async (signins: ISignin[]) => {
   await browserStorageService.setValue("signins", signinsObj);
 };
 
-export const getSigninsByDomain = async (url?: string) => {
+export const getDomainSignins = async (url?: string) => {
   if (!url) return [];
 
   const domain = getDomainFromUrl(url);
   const signins = await getSignins();
   return signins?.filter((signin) => signin.domain === domain);
 };
 
-export const updateAutoSigninByDomain = async (signin: ISignin) => {
+export const getDomainSigninByIssueName = async (
```

Comment:
recommend that the name say 'issuee' making it getDomainSigninByIssueeName because someday we may have separate methods for issuer/issuee and it will be hard to distinguish.